database/omni_db_schema.psql

```diff
@@ -319,25 +319,11 @@ create table if not exists OfferAccepts (
 create index Buyers on OfferAccepts (Buyer)
 ;
 
-/* directional exchange rates between pairs of properties; can work with fiat currencies as well */
-/* rate for 1 --> 2 not necessarily the same as the reciprocal of rate for 2 --> 1 */
-create table if not exists ExchangeRates (			
-	Protocol1 Protocol		/* see Protocols */
-	, PropertyID1 bigint		/* need exchange rates for fiat currencies */
-	, Protocol2 Protocol
-	, PropertyID2 bigint			
-	, Rate1For2 float		/* (1 for 2) */		
-	, AsOf timestamp(0)
-	, Source varchar(256)		/* URL */
-
-	, primary key (PropertyID1, Protocol1, PropertyID2, Protocol2) 
-)
-;
 
```

Comment:
If you're curious why we deleted this section, it's because we somehow ended up with a duplicate of the same table name
this is a duplicate of https://github.com/OmniLayer/omniEngine/pull/73/files#diff-2e8474eb6a8949054f3130d8badd8f2bR263